### PR TITLE
Added typing information for the state_change module

### PR DESCRIPTION
**Pull request by hackaugusto**

### CODE REVIEW

File: raiden/transfer/state_change.py [hunk 1]
```diff
@@ -376,7 +397,11 @@ def __ne__(self, other):
 class ActionChangeNodeNetworkState(StateChange):
     """ The network state of `node_address` changed. """
 
-    def __init__(self, node_address: typing.Address, network_state):
+    def __init__(
+            self,
+            node_address: typing.Address,
+            network_state,
```
[the remark this answers is not in the document]
Comment (hackaugusto) [reply raiden/transfer/state_change.py]: laziness on my part, the network_state is one of these three values:

https://github.com/raiden-network/raiden/blob/6ae3fc37773d75ddae0d5b22007d0058bcb832fd/raiden/transfer/state.py#L41-L43

so, I would have to import them in the typing module, and define a type with them. I wondered if that would introduced import cycles and just delayed it a bit

[the remark this answers is not in the document]
Comment (palango) [reply raiden/transfer/state_change.py]: Fine for me, we can add this later.

**Comment by LefterisJP** [comment]
@hackaugusto The typing information is fine but as I also told to palango, let's please not add the redundant `-> None` to each constructor. It's completely useless information and mypy will soon (hopefully) [fix it](https://github.com/python/mypy/issues/604#issuecomment-390024980) and not require it.

**Comment by hackaugusto** [comment]
@LefterisJP I didn't know that this design decision had changed, I read about it before 1/Dec. I will not add it for the future PRs

**Pull request merged**